src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  function add(...args) {
```

Comment:
The `add` function is using rest parameters to gather arguments, but according to the task description, the `adder` function should only ever receive one argument at a time. You should remove the rest parameter and use a single parameter instead.